services/web/server/src/simcore_service_webserver/login/storage.py

```diff
@@ -98,7 +98,8 @@ async def get_confirmation(self, filter_dict) -> ConfirmationTokenDict | None:
             filter_dict["user_id"] = filter_dict.pop("user")["id"]
         async with self.pool.acquire() as conn:
             confirmation = await _sql.find_one(conn, self.confirm_tbl, filter_dict)
-            return ConfirmationTokenDict(**confirmation) if confirmation else None
+            confirmation_token: ConfirmationTokenDict | None = ConfirmationTokenDict(**confirmation) if confirmation else None  # type: ignore[misc]
```

Comment:
how is this not getting reformatted? line seems a bit too long to me

Reply:
hmm seems the type ignore was blocking the formatting, I will push change